Makefile

```diff
@@ -7,20 +7,24 @@ PATH := bin:$(PATH)
 TMPDIR := $(shell mktemp -d)
 
 .PHONY: all
-all: doc binary debug test check
+all: doc minimal-binary debug test check
+
+.PHONY: minimal-binary
+minimal-binary: doc
+	go build -tags minimal -v  -ldflags "-X  github.com/anuvu/zot/pkg/api.Commit=${COMMIT}" -o bin/zot-minimal ./cmd/zot
 
 .PHONY: binary
 binary: doc
-	go build -v -ldflags "-X  github.com/anuvu/zot/pkg/api.Commit=${COMMIT}" -o bin/zot -tags=jsoniter ./cmd/zot
+	go build -tags extended -v -ldflags "-X  github.com/anuvu/zot/pkg/api.Commit=${COMMIT}" -o bin/zot ./cmd/zot
```

Comment:
s/extended/extensions/g

Reply:
Should I used tags as extended/extensions ??